src/election_anomaly/db_routines/__init__.py

```diff
@@ -16,6 +18,9 @@
 import os
 
 
+class DatabaseError(psycopg2.OperationalError):
```

Comment:
I did a grep and couldn't find anywhere where this is used.

Reply:
That's strange about "Checking cdf_element.txt", etc. I can't find the lines, and I don't see those lines when I run the program on this branch.

Reply:
Turns out that was user error